### ISSUE TITLE
VTModuleDesigner installation issue in vtiger 6.3

### ISSUE THREAD

**Issue by jaga8693** [sardoj/VTModuleDesigner#132]
Hi,
I am new to vtigercrm, vtigercrm is really superb. I have downloaded the vtiger 6.3 version and working on it now am trying to add VTModuleDesigner using settings->studio->Module Manager. I have uploaded the zip file which i have downloaded but while installing it shows error message as

dUnzip2: File 'manifest.xml' is not compressed in the zip.
Warning: simplexml_load_file(): I/O warning : failed to load external entity "test/vtlib/manifest-1448859577.xml" in C:\xampp\htdocs\finance\vtlib\Vtiger\PackageImport.php on line 54
Warning: unlink(test/vtlib/manifest-1448859577.xml): No such file or directory in C:\xampp\htdocs\finance\vtlib\Vtiger\PackageImport.php on line 55
FailedInvalid File provided for module import! Try Again.

Can you help me out of it..

**Comment by jaga8693** [comment]
Thanks. I have installed module designer successfully by downloading a new file which you have posted in someone issue this is the file name which I downloaded "ModuleDesigner_2014-11-09".

I have created a module and tried to add some values in it.. Values get stored in my db but blank page is showing. Url redirect to http://sitedomain/index.php. May i know why this happens, Not only for inserting values for updating the same issue occurs

Thanks.
